doc/md_docs_test.go

```diff
@@ -40,6 +42,7 @@ func TestGenMdDoc(t *testing.T) {
 	checkStringContains(t, output, echoSubCmd.Short)
 	checkStringOmits(t, output, deprecatedCmd.Short)
 	checkStringContains(t, output, "Options inherited from parent commands")
+	checkStringOmitsSuffix(t, output, doubleLineBreak)
```

Comment:
This is nice, but don't we want the more general check, that the output always ends with a single newline?